src/cmdc_tools/datasets/official/NV/data.py

```diff
@@ -0,0 +1,287 @@
+
+            await graph.click(button="right")
+            # Get table button
+            table_button = await page.waitForXPath("//h6[text()='Show as a table']")
+            await table_button.click()
+
+            labels = await self._get_labels_from_graph(page)
+            # parse labels
+            data = {"Suspected": [], "Confirmed": []}
+            for label in labels:
+                split = label.split(". ")
+                date = split[0].split("Date")[1].strip()
+                tests = split[1].split(" ")
+                tests_type = tests[0].strip()
+                # Skip all new tests
+
+                tests_num = int(tests[1][:-1].strip().replace(",", ""))
+                {"Date": date, f"{tests_type}": tests_num}
+
+                # data.append({"Date": date, f"{tests_type}": tests_num})
+                data[tests_type].append({"dt": date, "value": tests_num})
+            suspected = pd.DataFrame(data["Suspected"]).assign(
+                variable_name="hospital_beds_in_use_covid_suspected"
+            )
+            confirmed = pd.DataFrame(data["Confirmed"]).assign(
+                variable_name="hospital_beds_in_use_covid_confirmed"
+            )
+
+            df = pd.concat([suspected, confirmed], sort=False, ignore_index=True)
+
+            df.dt = pd.to_datetime(df.dt)
+            return df.assign(vintage=pd.Timestamp.utcnow(), fips=self.state_fips)
+
+    def _parse_hospitalizations(self, labels):
+        pass
+
+
+class NevadaCounty(DatasetBaseNoDate):
+    state_fips = int(us.states.lookup("Nevada").fips)
+    has_fips = False
+    source = "https://app.powerbigov.us/view?r=eyJrIjoiMjA2ZThiOWUtM2FlNS00MGY5LWFmYjUtNmQwNTQ3Nzg5N2I2IiwidCI6ImU0YTM0MGU2LWI4OWUtNGU2OC04ZWFhLTE1NDRkMjcwMzk4MCJ9"
+
+    def get(self):
+        return asyncio.run(self._get_county_overview_async())
+
+    async def _get_county_overview_async(self):
+        async with with_page() as page:
+            await page.goto(self.source)
+            # Wait for dashboard to load
+            # Get next page button
+            await page.waitForXPath("//span[text()='COVID-19 ']")
+            button = await _get_next_page_button(page)
+            await button.click()
+            # Wait for dashboard to load
+            # Go to next page
+            await page.waitForXPath("//div[text()='COVID-19 Statistics by County']")
+
+            table = await page.waitForXPath("//div[@class='tableExContainer']")
+
+            col_headers = await _get_table_vals(page, table, "columnHeaders")
+
+            body_cells = np.array(await _get_table_vals(page, table, "bodyCells"))
+
+            out = pd.DataFrame(
+                body_cells.reshape((-1, len(col_headers)), order="F"),
+                columns=col_headers,
+            )
+
+            renamed = out.rename(
+                columns={
+                    "County": "county",
+                    "Tests": "tests_total",
```

Comment:
Can we collect the "People Tested" column rather than the "Tests" column -- We prefer to report the number of people tested rather than the number of tests administered.